cantools/subparsers/generate_c_source.py

```diff
@@ -81,6 +92,10 @@ def add_subparser(subparsers):
         '-f', '--generate-fuzzer',
         action='store_true',
         help='Also generate fuzzer source code.')
+    generate_c_source_parser.add_argument(
+        '-o', '--output-directory',
+        default='.',
+        help='Directory in which to write output files.')
```

Comment:
Change to `help='Directory in which to write output files (default: %(default)s).')`.

Reply:
Sounds like you actually want to update to use the [`ArgumentDefaultsHelpFormatter`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter).

Reply:
Not in this PR, maybe later.

Reply:
That is what I meant, yes.

Reply:
See #275.